app/models/maintenance_tasks/task_data.rb

```diff
@@ -135,7 +135,13 @@ def csv_task?
     private
 
     def runs
-      Run.where(task_name: name).with_attached_csv_file.order(created_at: :desc)
+      if ActiveStorage::Attachment.table_exists?
```

Comment:
Can we keep this logic inside Run? Maybe overriding `with_attached_csv_file`?

Reply:
Actually overriding `with_attached_csv_file` involves overriding the scope, which complains noisily when I try to do it. I'll create our own scope instead, let me know what you think.